pfcpiface/bess.go

```diff
@@ -31,7 +31,11 @@ func (b *bess) setInfo(udpConn *net.UDPConn, udpAddr net.Addr, pconn *PFCPConn)
 	log.Println("bess setUdpConn not handled")
 }
 
-func (b *bess) sendMsgToUPF(method string, pdrs []pdr, fars []far) uint8 {
+func (b *bess) handleChannelStatus(u *upf) bool {
+	return false
```

Comment:
implement the logic to check if grpc channel is connected or not

Reply:
done